packages/client/src/views/SysAdmin/Config/DynamicModal.tsx

```diff
@@ -0,0 +1,242 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import { PrimaryButton, TertiaryButton } from '@opencrvs/components/lib/buttons'
+import {
+  NOTIFICATION_TYPE,
+  ResponsiveModal
+} from '@opencrvs/components/lib/interface'
+import { buttonMessages } from '@client/i18n/messages'
+import { messages } from '@client/i18n/messages/views/config'
+import * as React from 'react'
+import { injectIntl, WrappedComponentProps as IntlShapeProps } from 'react-intl'
+import { connect } from 'react-redux'
+import styled from '@client/styledComponents'
+import { InputField, TextInput } from '@opencrvs/components/lib/forms'
+import { GeneralActionId } from '@client/views/SysAdmin/Config/Application'
+import { EMPTY_STRING } from '@client/utils/constants'
+import { Alert } from '@opencrvs/components/lib/icons/Alert'
+import { configApplicationMutations } from '@client/views/SysAdmin/Config/mutations'
+import { updateOfflineConfigData } from '@client/offline/actions'
+
+const Message = styled.div`
+  margin-bottom: 16px;
+`
+const ApplyButton = styled(PrimaryButton)`
+  height: 40px;
+  & div {
+    padding: 0 8px;
+  }
+`
+const CancelButton = styled(TertiaryButton)`
+  height: 40px;
+  & div {
+    padding: 0;
+  }
+`
+const Content = styled.div`
+  display: flex;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    flex-direction: column-reverse;
+  }
+`
+const ErrorContent = styled.div`
+  display: flex;
+  margin-bottom: 20px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    flex-direction: column-reverse;
+  }
+`
+const Field = styled.div`
+  margin-bottom: 30px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    margin-bottom: 0px;
+  }
+`
+const HalfWidthInput = styled(TextInput)`
+  width: 271px;
```

Comment:
Why is it 271px?

Reply:
I have missout the actual size. Changed it to 300px as suggested by Jonathan.